enterprise/security/src/main/java/org/neo4j/server/security/enterprise/auth/AuthProcedures.java

```diff
@@ -77,6 +77,8 @@ else if ( !enterpriseSubject.isAdmin() )
         else
         {
             enterpriseSubject.getUserManager().setUserPassword( username, newPassword );
+            terminateTransactionsForValidUser( username );
+            terminateSessionsForValidUser( username );
```

Comment:
Is it not enough to terminate all sessions since that will also terminate active transactions?